docs/sql-ref-syntax-qry-select.md

```diff
@@ -43,11 +43,23 @@ While `select_statement` is defined as
 ```sql
 SELECT [ hints , ... ] [ ALL | DISTINCT ] { named_expression [ , ... ] }
     FROM { from_item [ , ...] }
+    [ PIVOT clause ]
+    [ LATERAL VIEW clause ] [ ... ] 
     [ WHERE boolean_expression ]
     [ GROUP BY expression [ , ...] ]
     [ HAVING boolean_expression ]
 ```
 
+While `PIVOT clause` is defined as
+```
+PIVOT ( { group_expression [ AS group_expression_alias ] } [ , ... ] FOR column_list IN ( expression_list ) )
+```
+
+While `LATERAL VIEW clause` is defined as
+```
+LATERAL VIEW [ OUTER ] { generating_function (generating_function_expression) [ table_function_alias ] AS column_alias [ , ... ] } [ ... ]
+```
+
```

Comment:
We don't need this part because we already have the dedicated pages for the syntaxes.